### PR TITLE
chore: migrate bulkDelete to v5

### PR DESCRIPTION
**Pull request by Feranchz**
### What does it do?

Migrates BulkDelete to works with v5 and enabled some API tests related to BulkDelete

### CODE REVIEW

File: packages/core/content-manager/server/src/controllers/collection-types.ts [hunk 1]
```diff
@@ -598,18 +598,32 @@ export default {
...
-      filters: {
-        $and: [idsWhereClause].concat(permissionQuery.filters || []),
-      },
-    };
+    const { locale } = getDocumentLocaleAndStatus(body);
+
+    const documentLocales = await documentManager.findLocales(documentIds, model, {
+      populate,
+      locale,
+    });
```
Comment (Marc-Roig) [inline packages/core/content-manager/server/src/controllers/collection-types.ts:611]: If you are only deleting one locale at a time, is it necessary to get multiple document locales here?

Side comment, you might need to update the typing of findLocales to accept an array of strings.

**Review by Marc-Roig — APPROVED** [review]

File: packages/core/content-manager/server/src/controllers/collection-types.ts (hunk 1 above)
Comment (Feranchz) [reply packages/core/content-manager/server/src/controllers/collection-types.ts:611]: I'm trying to think in future here 🤔 to when we need to implement the bulkDelete for multiple locales

But right now it works just like a `findMany`, do you think makes sense to keep it with findLocales or is better use findMany? (I guess is the same at the end)

Comment (Marc-Roig) [reply packages/core/content-manager/server/src/controllers/collection-types.ts:611]: I don't have strong opnions here. I normally try to "avoid" implementing before any feature has been decided, to reduce complexity in code, but here it is just chanigng findMany to findLocales so I don't see any issue on using this  :)

File: tests/api/plugins/i18n/content-manager/crud.test.api.js [hunk 2]
```diff
@@ -38,6 +38,37 @@ const data = {
...
+    qs: {
+      locale,
+    },
+    body: {
+      name,
+      locale,
+    },
+  });
+
+  console.log(res.body);
```
Comment (markkaylor) [inline tests/api/plugins/i18n/content-manager/crud.test.api.js:67]: 👀

**Review by jhoward1994 — APPROVED** [review]
Looks good. I don't think much (or any 🤞🏻) of this will conflict with the bulk locale actions

**Pull request merged**